modules/generator/registry/gauge.go

```diff
@@ -164,10 +165,20 @@ func (g *gauge) removeStaleSeries(staleTimeMs int64) {
 	g.seriesMtx.Lock()
 	defer g.seriesMtx.Unlock()
 
+	expTimeMs := staleTimeMs
```

Comment:
not psyched about this custom expiration logic. we run with default stale duration of 15m, but we could likely bring that down.

Reply:
Given that each metric type already had separate implementations to remove stale series, this seemed like it would have less overall impact than reducing stale duration across the board. The logic should remain the same unless `SetExpiration` is called explicitly on the gauge. 

If it's more desirable to lower stale duration for all metrics, I can remove this and we can look into the impact of that update.

Reply:
yeah, i just don't like that all series are expired in the same way on the same cutoff time except for a few gauges that internally calc their own cutoff time based on time.Now()

this feels more like a business decision then a technical one. can we not do what we need to do with a standard stale duration? perhaps we should sync offline

Reply:
thanks for the feedback Joe. I removed the custom expiration logic